fbfmaproom/data-conversion-scripts/oromia/bad-years-v3-3season.py

```diff
@@ -0,0 +1,154 @@
+import cftime
```

Comment:
This file should be moved from the oromia directory to the ethiopia directory.

Since it also handles data for djibout, let's put a git-managed symlink to it from the djibouti directory. And then after we get this merged, we can look into creating a library of reusable data conversion utility functions.

Reply:
done, I moved the script to ethiopia , then -> git symbolic-ref bad-years-v3-3season.py ../djibouti/bad-years-v3-3season.py